sr/comp/mixtape/cli.py

```diff
@@ -81,6 +81,7 @@ def play(args):
             config['password'],
             config['source_name'],
             config['scene_name'],
+            config.get('preroll_time', 2),
```

Comment:
Could we make this non-optional?
```suggestion
            config['preroll_time'],
```

Reply:
Certainly, it was only optional for backward compatibility but given the feature is only 3 days old we don't really need to support that.

Reply:
Yeah, I'm mixed on backwards compatibility at all for this sort of project. It's somewhat unlikely that anyone would be using this other than us and in any case it's very much a tool _for us_ and each use of it is pretty much a one-shot. I think the benefits of simpler code and (in this case) forcing it to be explicit outweigh the benefits of compatibility.